app/v2/controllers/V2MovementsController.scala

```diff
@@ -351,15 +346,32 @@ class V2MovementsControllerImpl @Inject() (
             (for {
               downloadUrl   <- handleUpscanSuccessResponse(upscanResponse)
               objectSummary <- objectStoreService.addMessage(downloadUrl, movementId, messageId).asPresentation
+
               messageUpdate = MessageUpdate(MessageStatus.Processing, Some(ObjectStoreURI(objectSummary.location.asUri)))
-              update <- persistenceService.updateMessage(movementId, messageId, messageUpdate).asPresentation
-            } yield update).fold[Result](
+              _ <- persistenceService.updateMessage(eori, movementType, movementId, messageId, messageUpdate).asPresentation
+              messageType = extractMessageType(movementType)
```

Comment:
Can we add a TODO here to properly extract the message type later? I get why we've not done it now, but it can't be forgotten.